flows/actions/testdata/call_classifier.json

```diff
@@ -104,6 +104,14 @@
     },
     {
         "description": "Result with category success created if classification happens",
+        "http_mocks": {
```

Comment:
testing these with real wit client with mocked HTTP calls to better test recording of non-successful HTTP requests